projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-control-model-base.ts

```diff
@@ -0,0 +1,48 @@
+import { TableControlOption, TableControlOptionType } from '@hypertrace/components';
+import { BOOLEAN_PROPERTY, ModelApi, ModelProperty } from '@hypertrace/hyperdash';
+import { ModelInject, MODEL_API } from '@hypertrace/hyperdash-angular';
+import { uniqWith } from 'lodash-es';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+export abstract class TableWidgetControlModelBase {
```

Comment:
naming preference nit: `TableWidgetControlModel` (I just hate using `*Base`, `Abstract*`, `*Impl` and the like)